source/shared/core_sqlsrv.h

```diff
@@ -997,6 +1045,8 @@ struct sqlsrv_conn : public sqlsrv_context {
     sqlsrv_conn( SQLHANDLE h, error_callback e, void* drv, SQLSRV_ENCODING encoding  TSRMLS_DC ) :
         sqlsrv_context( h, SQL_HANDLE_DBC, e, drv, encoding )
     {
+        server_version = SERVER_VERSION_UNKNOWN;
+        driver_version = ODBC_DRIVER_13; 
```

Comment:
Is it a good idea to set default to ODBC_DRIVER_13? What about ODBC 11?
How about DRIVER_VERSION_UNKNOWN or something like that?